ChangeLog

```diff
@@ -488,6 +488,9 @@ New deprecations
      deprecated.
    * PSA_ALG_AEAD_WITH_DEFAULT_TAG_LENGTH and PSA_ALG_AEAD_WITH_TAG_LENGTH
      have been renamed, and the old names deprecated.
+   * Signed magnitude and ones' complement implementations for signed integers are
+     no longer supported. Two’s Complement is the only supported
```

Comment:
I'm not sure if “no longer supported” is completely accurate: we tried to support them, but we aren't sure we actually did.

Reply:
Yes, I think "are now officially not supported" would be more accurate.

Reply:
I agree, changing the wording as stated by @mpg